lib/presentation/widget/upload_request_group/upload_request_group_widget.dart

```diff
@@ -513,4 +514,20 @@ class _UploadRequestGroupWidgetState extends State<UploadRequestGroupWidget> {
                           _buildActiveClosedUploadTile(uploadRequestList[index]))))
     ]);
   }
+
+  List<Widget> _contextMenuActionTiles(BuildContext context, UploadRequestGroup uploadRequestGroup) {
+    return [
+      _AddRecipientsAction(context, uploadRequestGroup),
+    ];
+  }
+
+  Widget _AddRecipientsAction(BuildContext context, UploadRequestGroup uploadRequestGroup) {
+    return SimpleContextMenuActionBuilder(
+            Key('add_recipients_context_menu_action'),
+            SvgPicture.asset(imagePath.icAddMember,
+                width: 24, height: 24, fit: BoxFit.fill),
+            AppLocalizations.of(context).add_recipients)
+        .onActionClick((_) => _model.goToAddRecipients(uploadRequestGroup))
```

Comment:
Need to check UploadRequestGroup state here. That's not all UploadRequestGroup can add recipient

Reply:
It's not specified on the UC, which upload request group can not add pls ? I thought it was collective or not but in the web version they both can

Reply:
<img width="782" alt="Screen Shot 2021-08-17 at 10 41 48" src="https://user-images.githubusercontent.com/29337364/129660142-e7e3ea15-06cf-44ef-9607-41a1988476be.png">
 
You can easily to see that from web version. As I checked, the UR can not add more recipients when it status is Expired/Closed or Archieved. Let's check again carefully!